Source/mainmenu.cpp

```diff
@@ -155,10 +155,20 @@ BOOL mainmenu_init_menu(int type)
 	return success;
 }
 
+#if NONET
+void UiSelOkDialog(const char *title, const char *body, bool background);
+#endif
+
 BOOL mainmenu_multi_player()
 {
 	gbMaxPlayers = MAX_PLRS;
+
+#if NONET
+	UiSelOkDialog(NULL, "There is no network support in this Xbox port at the momment...sorry", false);

```

Comment:
Is the Xbox not able to run the loopback multiplayer option?

Reply:
I think the Xbox can do loopback multiplayer I was just focused on the single player experience at the time.

No worries, I should be able to get the multiplayer dialogs converted this weekend.

Reply:
Sounds good, I will focuse on other areas then.